state/upgrades.go

```diff
@@ -2942,6 +2939,9 @@ func ReplaceNeverSetWithUnset(pool *StatePool) (err error) {
 				Update: update,
 			})
 		}
+		if err := iter.Close(); err != nil {
```

Comment:
I think most other places we just do a 
```go
defer func() { _ = iter.Close() }()
``` 
after opening.

Reply:
I was wrong anyway. We check nearly everywhere, but we have defer calls in addition that I don't think we need.